### DISCUSSION TITLE
夜间模式中间部分变白如何解决

### DISCUSSION THREAD

**Discussion by Anthonyuncle** [the1812/Bilibili-Evolved#2601] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/65503687/145028194-406ea7d7-cc23-4480-a68a-779adf07701a.png)

**Comment by the1812** [comment, marked as answer]
见 #2597

**Reply by littleButterflies** [reply]
我也是今晚打卡播放页发现这个问题了，经过浏览器元素检测，选择器 【.v-warp】对应元素 中间内容容器 B站 强制添加了 纯白背景色 background-color:#fff;

**Reply by ChaXxl** [reply]
这个有很多人问过了。。。
v2.0 preview版本已经修复夜间模式的问题了
